File: timeconsumingfunrand.py
import time
import random

def timeconsumingfun(arg):
    # Generate a random integer between 1 and 5
    pause_duration = random.randint(1, 5) 
    print(f'Pausing for {pause_duration} seconds...')  # Display the pause duration
    time.sleep(pause_duration)  # Pause for the random duration

# import time
# def timeconsumingfun(arg):
#     time.sleep(5)

File: timeconsumingfunrand.m
function timeconsumingfun
    % Generate a random integer between 1 and 5
    pauseDuration = randi([1, 5]); 
    fprintf('Pausing for %d seconds...\n', pauseDuration); % Display the pause duration
    pause(pauseDuration);           % Pause for the random duration
end
% function timeconsumingfun
%     pause(5)
% end